e2e/cypress/e2e/add-todo.cy.js

```diff
@@ -0,0 +1,52 @@
+describe("Add Task Action", () => {
+   beforeEach(() => {
+      cy.visit("http://localhost:3000");
+      cy.get(".HomeCard_clear-all__Rq0JP").click();
```

Comment:
try to make classes like this as constants:
HOME_CLEAL_ALL_CLASS_NAME = "HomeCard_clear-all__Rq0JP" for example.
When having all these constants you can reuse them in many tests and it make the test much more clear and readable

Reply:
Right. On it, thanks!